store/src/db/mod.rs

```diff
@@ -49,25 +49,33 @@ impl Db {
             create_dir_all(p)?;
         }
 
-        let pool =
-            SqliteConfig::new(config.database_filepath.clone()).create_pool(Runtime::Tokio1)?;
-
-        let conn = pool.get().await?;
+        let pool = SqliteConfig::new(config.database_filepath.clone())
+            .builder(Runtime::Tokio1)?
+            .post_create(Hook::async_fn(move |conn, _| {
+                Box::pin(async move {
+                    // Feature used to support `IN` and `NOT IN` queries
+                    let _ = conn
+                        .interact(|conn| array::load_module(conn))
+                        .await
+                        .map_err(|_| HookError::StaticMessage("Loading carray module failed"))?;
+
+                    let _ = conn
+                        .interact(|conn| migrations::MIGRATIONS.to_latest(conn))
+                        .await
+                        .map_err(|_| {
+                            HookError::StaticMessage("Migration task failed with a panic")
+                        })?;
```

Comment:
This probably doesn't need to happen for every connection - if so, we should keep it the way it was.

Reply:
Agreed, changed it so it runs on one connection.